src/docs/reference/modules/Resources/README.md

```diff
@@ -365,6 +369,23 @@ The style block will only be injected once based on its name and can optionally
     }
 </style>
 ```
+#### Link tag
+
+A link tag is used to define the relationship between the current document and an external resource such as a a favicon or stylesheet. For a stylesheet, however, use the [style helper](#register-a-named-script). 
+
+``` liquid tab="Liquid"
+{% link rel:"icon", type:"image/png", sizes:"16x16", src:"~/media/favicon/favicon-16x16.png" %}
```

Comment:
Cool, but don't use this `~/media/favicon/favicon-16x16.png` to get to the media library folders, as it doesn't take into account any tenant prefix.

`~/TheTheme` the `~` prefix will get you to a static file hosted in a `wwwroot` folder, but not a media folder, which is stored in the `App_Data` (so would work, but not across tenants)

For accessing the media library with any of the script / link tag helpers you would have to use the `@Html.AssetUrl` helper inside the `src` attribute. And I'm not sure how we would do this with liquid, it maybe something we are missing.

We might need to target `asset-src` on the `LinkTagHelper` and map the public url to the source url.

Reply:
@deanmarcussen no problem I can change these urls.

I didn't want to use static assets in my theme for favicon as you would want to change this if the theme is used by more than one tenant. Ideally you would have this in a custom setting but in my case I was under the pump to get it done so I was taking shortcuts.

I am going to raise an issue actually as I found a couple of problems in the link tag helper:

- The liquid asset_url helper which I wanted to use in my case to refer to a media file doesn't work inside the link tag helper
- the link tag helper works from within liquid field, eg. from a layer, but if the tag is in a static liquid template it doesn't work - no idea why
- The razor version of the helper doesn't work at all actually if my code example is right. The example in the documentation I added just renders the link exactly as is in the exact spot it is placed within a template. Now that I am writing this i am thinking it might be that I was missing the using statement - will check
- link helper doesn't seem to support named resources either which would be handy.

I will just check whether I had the using and will log these issues.

Reply:
I have tested the razor helper again and the issue was due to the helper reference missing.  I also re-tested the liquid helper and now the only outstanding issue is that you can't use the asset_url helper.